### PR TITLE
CLI --verbose flag

### PR DESCRIPTION
**Pull request by jayvdb**
Also eplaced `eprintln` with `log` in `butane_test_helper`.

### CODE REVIEW

File: examples/getting_started/src/butane_migrations.rs [hunk 1]
```diff
@@ -157,7 +157,7 @@ pub fn get_migrations() -> Result<MemMigrations, butane::Error> {
       },
       "from": null,
       "up": {
-        "pg": "CREATE TABLE Blog (\nid BIGSERIAL NOT NULL PRIMARY KEY,\n\"name\" TEXT NOT NULL\n);\nCREATE TABLE Post (\nid SERIAL NOT NULL PRIMARY KEY,\ntitle TEXT NOT NULL,\nbody TEXT NOT NULL,\npublished BOOLEAN NOT NULL,\nblog BIGINT NOT NULL,\nbyline TEXT \n);\nCREATE TABLE Post_tags_Many (\nowner INTEGER NOT NULL,\nhas TEXT NOT NULL\n);\nCREATE TABLE Tag (\ntag TEXT NOT NULL PRIMARY KEY\n);\n",
+        "pg": "CREATE TABLE Blog (\nid BIGSERIAL NOT NULL PRIMARY KEY,\n\"name\" TEXT NOT NULL\n);\nCREATE TABLE Post (\nid SERIAL NOT NULL PRIMARY KEY,\ntitle TEXT NOT NULL,\nbody TEXT NOT NULL,\npublished BOOLEAN NOT NULL,\nblog BIGINT NOT NULL,\nbyline TEXT \n);\nCREATE TABLE Post_tags_Many (\nowner INTEGER NOT NULL,\nhas TEXT NOT NULL\n);\nCREATE TABLE Tag (\ntag TEXT NOT NULL PRIMARY KEY\n);\nCREATE TABLE IF NOT EXISTS butane_migrations (\n\"name\" TEXT NOT NULL PRIMARY KEY\n);\n",
```
Comment (Electron100) [inline examples/getting_started/src/butane_migrations.rs:160]: Did you mean to include this file? I wouldn't have expected a change here from the CLI verbosity...

**Review by Electron100 — CHANGES_REQUESTED** [review]

File: examples/getting_started/src/butane_migrations.rs (hunk 1 above)
Comment (jayvdb) [reply examples/getting_started/src/butane_migrations.rs:160]: This change has already been merged as part of https://github.com/Electron100/butane/pull/206

Comment (jayvdb) [reply examples/getting_started/src/butane_migrations.rs:160]: It was a bug in the "backend add/remove" PR

Comment (Electron100) [reply examples/getting_started/src/butane_migrations.rs:160]: Ah, I see

**Review by Electron100 — APPROVED** [review]

**Pull request merged**